mysql-test/t/cte_recursive.test

```diff
@@ -1304,6 +1304,7 @@ drop table folks;
 create table t1(a int);
 insert into t1 values (0),(1),(2),(3),(4),(5),(6),(7),(8),(9);
  
+--source include/analyze-format.inc
```

Comment:
I'm wondering why is this necessary here?  The statement is `EXPLAIN`, not `ANALYZE`.  EXPLAIN does not produce any execution-time members (those that start with `r_`, like r_total_time_ms or r_buffer_size).   
That fact that this patch doesn't change the r/cte_recursive.result file proves this point